app/views/derivatives/_derived_from.html.erb

```diff
@@ -0,0 +1,9 @@
+<div class="items-heading mt-0 mb-0">
+  <h2 class="mt-0 mb-0">Codes derived from this code</h2>
+</div>
+
+<blockquote>
+  These codes use <strong><%= @post.code %></strong> as part of their code. This could be because they built off of it, used parts of it, used it for inspiration, or other reasons.
```

Comment:
Same down here. I think "These posts use CODE as part of their code. etc. etc. etc." sounds much better.

Reply:
Also, a bit of wordsmithing, but I think
```
This could be because they built off of it, used parts of it, used it for inspiration, or other reasons.
```
could be changed to
```
This could be because they remixed it, used parts of it, were inspired by it, or other reasons.
```